### DISCUSSION TITLE
Pico Pi USB Host audio

### DISCUSSION THREAD

**Discussion by paulhamsh** [cherry-embedded/CherryUSB#82] [General]
Hi
I really want to use a Pico Pi (rp2040) with USB Host audio - TinyUSB doesn't do that, and it looks like CherryUSB does - very exciting and amazing!
I couldn't see a demo for USB Host audio for any boards - is there one, and if not, where do I start on getting it to work with the Pico Pi?
Thanks

**Comment by sakumisu** [comment]
If the hcd port with iso transfer is supported, host audio can use, so only ehci and dwc2 have supported iso transfer now. RP2040 has not implemented hcd port.

**Comment by sakumisu** [comment]
If you have free time and ability to implement rp2040hcd port, maybe you can use all the classes.

**Reply by paulhamsh** [reply]
Ah ok - I don't have the ability to do that. I thought it was done already.

**Comment by sakumisu** [comment]
My friend will support for this, need sometime.
